setup.cfg

```diff
@@ -10,4 +10,4 @@ application-import-names = leapp
 # check only for correct group type and abc sorting without group
 import-order-style = pep8
 # default values + ignore strict alphabetical sorting within one module import
-ignore = E121,E123,E126,E133,E241,E242,E226,W503,W504,W505,I101
+ignore = E121,E123,E126,E133,E241,E242,E226,W503,W504,W505,I101,E741
```

Comment:
It depends. But in majority number of cases I will disagree. Using instead `line` instead of `l` is not big deal. As well, the code itself is really bad when you call `l` in block of code. The `line` seems much more clear so I prefer to drop this one change.

Reply:
Discussed in person. This error is raised on fedora but it is not raised in our CI (older version of linter in centos container, maybe?).. From that point, just create an issue, so we will fix it in future and put it to the ignore list for now.

Reply:
Done https://github.com/oamg/leapp-repository/issues/502 @pirat89